src/ks-app/src/app/containers/drag-and-drop/drag-and-drop.component.html

```diff
@@ -0,0 +1,179 @@
+<!--
+  ~ Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+  ~ This software is released under MIT license.
+  ~ The full license information can be found in LICENSE in the root directory of this project.
+  -->
+<h2>Drag and Drop</h2>
+
+<div class="clr-example">
+    <h4>Basic Draggable</h4>
+    <div class="row">
+        <div class="col-xs-12">
+            <p>
+                Use <code class="clr-code">clrDraggable</code> directive to make an element or component draggable.
+            </p>
+            <button class="btn btn-primary" clrDraggable>Draggable Button</button>
+            <button class="btn btn-success" clrDraggable>Draggable Button</button>
+            <button class="btn btn-warning" clrDraggable>Draggable Button</button>
+        </div>
+    </div>
+    <div class="row">
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 1</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 2</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+        <div class="col-xs-4">
+            <div class="card" clrDroppable>
+                <div class="card-block">
+                    <h3 class="card-title">Droppable Card 3</h3>
+                    <p class="card-text">
+                        ...
+                    </p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class="clr-example">
+    <h4>Basic Draggable</h4>
+    <div class="row">
+        <div class="col-xs-12">
+            <p>
+                Use <code class="clr-code">clrDragAndDropGroup</code> directive to make particular draggables interact
+                with particular droppables.
+            </p>
+            <button class="btn btn-primary" clrDraggable clrDragAndDropGroup="dnd-1">Draggable Button</button>
+            <button class="btn btn-success" clrDraggable clrDragAndDropGroup="dnd-2">Draggable Button</button>
+            <button class="btn btn-warning" clrDraggable clrDragAndDropGroup="dnd-3">Draggable Button</button>
```

Comment:
Can you try adding an array to the `clrDropGroup`?

Reply:
Yes! I have updated the app to demonstrate this case.